File: LV1/1.7/abc/abc.go
package main

import (
	"fmt"
	"sort"
)

func main() {
	l := make([]int, 3)
	var o string

	for i := 0; i < 3; i++ {
		fmt.Scanf("%d", &l[i])
	}
	fmt.Scanln(&o)
	sort.Ints(l)

	for _, c := range []rune(o) {
		fmt.Printf("%d ", l[c-'A'])
	}
}

File: LV1/1.7/abc/abc.cpp
#include <algorithm>
#include <cstdio>

int main() {
    int l[3];
    char o[3];

    for (int i = 0; i < 3; i++) {
        scanf("%d", &l[i]);
    }
    scanf("%s", o);

    std::sort(l, l + 3);

    for (int i = 0; i < 3; i++) {
        printf("%d ", l[o[i] - 'A']);
    }
}